File: src/parallel_parking/src/parallel_parking_node.cpp
#include "rclcpp/rclcpp.hpp"
class MyNode : public rclcpp::Node
{
public:
    MyNode() : Node("parallel_parking_node")
    {
        RCLCPP_INFO(this->get_logger(), "CPP Node has been started");
    }
private:
};
int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto node = std::make_shared<MyNode>();
    rclcpp::spin(node);
    rclcpp::shutdown();
}

File: src/parallel_parking/scripts/parallel_parking_node.py
#!/usr/bin/env python3

import rclpy
from rclpy.node import Node


class ParallelParkingNode(Node):
    def __init__(self):
        super().__init__("parallel_paking_node")
        self.get_logger().info("Python parallel_paking_node has been started.")

def main(args=None):
    rclpy.init(args=args)
    node = ParallelParkingNode()
    rclpy.spin(node)
    node.destroy_node()
    rclpy.shutdown()

if __name__ == "__main__":
    main()